pkg/kudoctl/packages/install/package.go

```diff
@@ -23,12 +24,16 @@ type Options struct {
 // Instance name, namespace and operator parameters are applied to the
 // operator package resources. These rendered resources are then created
 // on the Kubernetes cluster.
+// Packages can have dependencies on other packages. In that case,
+// dependent packages are resolved and their Operator and
+// Operatorversion resources created on the Kubernetes cluster.
 func Package(
 	client *kudo.Client,
 	instanceName string,
 	namespace string,
 	resources packages.Resources,
 	parameters map[string]string,
+	resolver resolver.Resolver,
```

Comment:
I wonder if it would work to rename some package so we get `resolver dependencies.Resolver` or `resolver packages.Resolver`?